readthedocs/builds/forms.py

```diff
@@ -37,3 +45,92 @@ def save(self, commit=True):
         if obj.active and not obj.built and not obj.uploaded:
             trigger_build(project=obj.project, version=obj)
         return obj
+
+
+class RegexAutomationRuleForm(forms.ModelForm):
+
+    match_arg = forms.CharField(
+        label='Custom match',
+        help_text=_(textwrap.dedent(
+            """
+            A
+            <a href="https://docs.readthedocs.io/page/automation-rules.html#user-defined-matches">
+                regular expression
+            </a>
+            to match the version.
```

Comment:
This is another user facing link that could use more context for screen readers/users if it's displayed to users.

Reply:
I have changed this text, but not sure if it's better